src/nvim/CMakeLists.txt

```diff
@@ -546,7 +546,7 @@ else()
 endif()
 set_target_properties(nvim_runtime_deps PROPERTIES FOLDER deps)
 
-file(COPY ${DEPS_PREFIX}/lib/nvim/parser DESTINATION ${PROJECT_BINARY_DIR}/lib/nvim/)
```

Comment:
It would be better to just make this conditional on the existence of `${DEPS_PREFIX}/lib/nvim/parser`.  "non-bundled" C parser can just be copied to anywhere in &rtp, it doesn't need any care by build system.